src/VectorStore.ts

```diff
@@ -320,14 +303,25 @@ export class VectorStore {
 		return new Map(dbFile.embeddings)
 	}
 
-	getNearestVectors(searchVector: Vector, resultNumber: number, relevanceThreshold: number): NearestVectorResult[] {
+	private similarityBatch(searchVectors: Vector[], possibleMatch: Vector): number {
```

Comment:
nit: Same comment as below, I think we can drop batch from the name it's intuitive from the array type.